db/migrate/20160603182320_add_en_passant_file_to_games.rb

```diff
@@ -0,0 +1,5 @@
+class AddEnPassantFileToGames < ActiveRecord::Migration
+  def change
+    add_column :games, :en_passant_file, :integer
```

Comment:
Lol. I see that [it is](https://github.com/letmecheck/app/pull/56/commits/d4fd906f54db0089d127b08b29fb66777660a6e1#diff-1b541d09ba44dabaa60af6e62f436119R61) 😄 It might be a good idea to record the strategy for computing whether or not it's a valid move in the commit message here, and something like an `#en_passant_eligible?` wrapping the validation logic involved.